sympy/matrices/matrices.py

```diff
@@ -2924,6 +2924,8 @@ def berkowitz(self):
         berkowitz_eigenvals
         """
         from sympy.matrices import zeros
+        if not self:
+            return (1,)
```

Comment:
It seems that the tuple returned by ``M.berkowitz()`` for an ``n x n`` matrix ``M`` is expected to have length ``n``. Why should it be different for an empty matrix (with n = 0)?